apps/atqr/api/src/app/filter/httpExceptions.ts

```diff
@@ -0,0 +1,45 @@
+import {
+  ExceptionFilter,
+  Catch,
+  ArgumentsHost,
+  HttpException,
+  InternalServerErrorException,
+} from '@nestjs/common';
+import { HttpAdapterHost } from '@nestjs/core';
+
+@Catch()
+export class HttpExceptionFilter implements ExceptionFilter {
+  constructor(private readonly httpAdapterHost: HttpAdapterHost) {}
+
+  catch(exception: HttpException, host: ArgumentsHost): string | void {
+    //catch expects receive HTTP exception and arguments of given by the handler
+    if (!(exception instanceof HttpException)) {
+      // if the exception is not HTTP one
+      console.log('You need handle this error');
+      console.log(
+        "Error handling example: throw new NotFoundException('we didn't find what you were looking for') "
+      );
+      console.log(exception);
+      exception = new InternalServerErrorException(
+        "We don't know what happen'd"
+      );
+    }
+    if (exception.message.length <= 10) {
+      console.log('Your message is too short. Do a better one');
+      exception = new InternalServerErrorException(
```

Comment:
The console log and error messages are contradictory. We know what happened, the message was to short, throw the appropriate error